src/core/commands.rs

```diff
@@ -70,6 +71,7 @@ impl PartialEq for Command {
 impl Debug for Command {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
+            Self::PersistAlternate(bool) => write!(f, "PersistAlternate({bool})"),
```

Comment:
Can you just give the inner value a different name than just calling it `bool`? It can be confusing down the road